engine/runtime/src/main/scala/org/enso/compiler/context/SuggestionDiff.scala

```diff
@@ -28,6 +32,20 @@ object SuggestionDiff {
           true
       }
 
+  case class SuggestionData(version : Int, suggestions : Iterable[Suggestion])
+
+  def deserialize(
+    json: String
+  ): Tree[Api.SuggestionUpdate] = {
+    val mapper  = new ObjectMapper() with ClassTagExtensions
+    mapper.registerModule(DefaultScalaModule)
+
+    val r : ObjectReader = mapper.readerFor[SuggestionData]
+    val o : SuggestionData = r.readValue[SuggestionData](json)
+    System.err.println("suggestions are here: " + o.suggestions)
```

Comment:
The suggestions are read properly. Now we only need to convert them into `Tree[Api.SuggestionUpdate]` somehow.